### PR TITLE
fix!: remove dialler language

### PR DESCRIPTION
**Pull request by achingbrain**
## Description

- Remove the confusing "dialler" word in metrics, replacing with "dial_queue"

### CODE REVIEW

**Comment by achingbrain** [comment]
@wemeetagain I've targeted the v1 release branch - please can you add some copy to the migration guide to cover these breaking changes.

**Comment by wemeetagain** [comment]
ready for re-review

**Comment by wemeetagain** [comment]
Would like to merge this first to avoid conflicts around adding a migration doc

**Comment by achingbrain** [comment]
Maybe open a PR with all the migration doc changes? That would avoid conflicts?

File: doc/migrations/v0.46-v1.0.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+```json
+
+```
+
+## Metrics
+
+The following metrics were renamed:
+
+`libp2p_dialler_pending_dials` => `libp2p_dialler_pending_dials`
```
Comment (maschad) [inline doc/migrations/v0.46-v1.0.md:52]:
```suggestion
`libp2p_dialler_pending_dials` => `libp2p_dial_queue_pending_dials`
```

File: doc/migrations/v0.46-v1.0.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+## Table of Contents <!-- omit in toc -->
+
+- [API](#api)
+- [Module Updates](#module-updates)
+- [Metrics](#metrics)
+
+## API
+
+<!--Describe breaking APIs with examples for Before and After
```
Comment (maschad) [inline doc/migrations/v0.46-v1.0.md:14]: Are these comments necessary?

**Review by maschad — CHANGES_REQUESTED** [review]

**Comment by maschad** [comment]
> Maybe open a PR with all the migration doc changes? That would avoid conflicts?

It may be hard to ensure all the changes are encompassed in the migration doc if it's a substantial diff.

File: doc/migrations/v0.46-v1.0.md (hunk 2 above)
Comment (wemeetagain) [reply doc/migrations/v0.46-v1.0.md:14]: They're from the template

**Review by maschad — APPROVED** [review]

File: doc/migrations/v0.46-v1.0.md (hunk 2 above)
Comment (wemeetagain) [reply doc/migrations/v0.46-v1.0.md:14]: I left these comments because other PRs will fill in these sections

**Comment by achingbrain** [comment]
This PR has added a `v0.46-v1.0.md` file when `v0.46-v1.0.0.md` already existed.

**Comment by wemeetagain** [comment]
Oops

**Pull request merged**